docker/install-deps

```diff
@@ -56,6 +56,18 @@ if [ -f /usr/bin/yum ]; then
         PKGLIST+="devtoolset-6-gcc-c++ devtoolset-6-binutils devtoolset-6-valgrind"
     fi
 
+    if [[ "$(rpm --eval %rhel)" = "7" ]]; then
+        until yum -y install centos-release-scl; do
+            echo "waiting"
+            sleep 1
+        done
+        PKGLIST=${PKGLIST/ gcc-c++/ devtoolset-7-gcc devtoolset-7-gcc-c++ devtoolset-7-binutils}
```

Comment:
it have no sense to add `devtoolset-7-gcc` it will be added as dependency of `devtoolset-7-gcc-c++`

Reply:
fixed